Ginger/Ginger/App.xaml.cs

```diff
@@ -23,6 +23,7 @@
 using Amdocs.Ginger.Common.InterfacesLib;
 using Amdocs.Ginger.Common.Repository;
 using Amdocs.Ginger.CoreNET.Repository;
+using Amdocs.Ginger.CoreNET.RosLynLib.Refrences;
```

Comment:
We want to reduce app items need to move

Reply:
Removed References from app